larray/tests/data/test2d.csv

```diff
@@ -1,4 +1,4 @@
-a\b,b0,b1,b2
-a0,0,1,2
-a1,3,4,5
-a2,6,7,8
+a\b,b0,b1
+1,0,1
```

Comment:
My comment about int-like labels was meant for the "horizontal" dimension, which is not handled by pandas but manually parsed by us. Having a test for int-like in a column is interesting too (to make sure Pandas does not change its handling of them) but less critical. In any case, we should have a test explicitly telling in a comment (or test name?) that this is what we test, so that we do not break it accidentally.

Reply:
If I add a new test array with 'int' labels for all axes, is that OK?

Reply:
sure!